auth-api/src/auth_api/services/notification.py

```diff
@@ -12,14 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Service for managing Invitation data."""
+import json
 
-from flask_mail import Message
+from flask import current_app
 
-from auth_api.extensions import mail
+from .rest_service import RestService
 
 
-def send_email(subject, sender, recipients, html_body):
+def send_email(subject: str, sender: str, recipients: str, html_body: str):  # pylint:disable=unused-argument
     """Send the email asynchronously, using the given details."""
-    msg = Message(subject, sender=sender, recipients=recipients.split())
-    msg.html = html_body
-    mail.send(msg)
+    notify_url = current_app.config.get('NOTIFY_API_URL') + '/notify/'
+    notify_body = {
+        'recipients': recipients,
+        'contents': {
+            'subject': subject,
+            'body': html_body
+        }
+    }
+    notify_response = RestService.post(notify_url, data=notify_body)
```

Comment:
Is notify open for public? Should we re-consider securing it with token from our realm?

Reply:
I don't think we open the notify service to public for now. In the Openshift environment, I pan to use service address instead of public route. Next step, we can think about it if need open. FastAPI support JWT token pretty easy.

Reply:
Ohh ok. If route is not exposed, all good!